source/fundamentals/crud/read-operations/change-streams.txt

```diff
@@ -288,3 +288,56 @@ output:
 
 For a list of options, see the `FullDocument <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/changestream/FullDocument.html>`__
 API documentation.
+
+Split Large Change Stream Events
+--------------------------------
+
+Starting in MongoDB 7.0, you can use a ``$changeStreamSplitLargeEvent``
+aggregation stage to split events that exceed 16 MB into smaller fragments.
+``$changeStreamSplitLargeEvent`` returns the fragments sequentially using the
+change stream cursor. Each fragment includes a ``SplitEvent`` object containing
+the following fields:
+
+.. list-table::
+   :header-rows: 1
+   :widths: 35 65
+
+   * - Field
+     - Description
+
+   * - ``fragment``
+     - The index of the fragment, starting at 1
+
+   * - ``of``
+     - The total number of fragments in the event
+
+Enable the ``$changeStreamSplitLargeEvent`` aggregation on your change stream as
+shown in the following example:
+
+.. code-block:: java
+
+   ChangeStreamIterable<Document> changeStream = collection.watch(
+         Arrays.asList(Document.parse("{ $changeStreamSplitLargeEvent: {} }")));
+
+.. note::
+
+   You can have only one ``$changeStreamSplitLargeEvent`` stage in your
+   aggregation pipeline, and it must be the last stage in the pipeline.
+
+You can access the ``SplitEvent`` by using your change stream cursor with the
+``getSplitEvent()`` as follows:
+
+.. code-block:: java
+
+   MongoChangeStreamCursor<ChangeStreamDocument<Document>> cursor = changeStream.cursor();
+   
+   // Application code that creates change stream events
+   // ...
+
+   SplitEvent event = cursor.tryNext().getSplitEvent();
+
+For more information about accessing data from a cursor, see :ref:`java-fundamentals-cursor`.
```

Comment:
Suggestion:

I think it might be useful to either qualify this information, given that it doesn't include specific information on change stream cursors, or omit it. To qualify the information, first it would be helpful to check what similarities the two different types of cursors have and then to include that.

Hypothetically, if MongoChangeStreamCursor extended MongoCursor, I would suggest something like the following:

The MongoChangeStreamCursor extends MongoCursor and shares the same methods to access data. To learn how to access data from a MongoCursor, see ...

Reply:
Removed this line since it's a ChangeStreamCursor not a MongoCursor